src/DashboardUI/src/pages/application/create/ApplicationCreatePage.tsx

```diff
@@ -0,0 +1,398 @@
+import { useNavigate } from 'react-router-dom';
+import MapProvider from '../../../contexts/MapProvider';
+import { ApplicationNavbar } from '../components/ApplicationNavbar';
+import { useConservationApplicationContext } from '../../../contexts/ConservationApplicationProvider';
+import Form from 'react-bootstrap/esm/Form';
+import { NotImplementedPlaceholder } from '../../../components/NotImplementedAlert';
+import FloatingLabel from 'react-bootstrap/esm/FloatingLabel';
+import Button from 'react-bootstrap/esm/Button';
+import { states } from '../../../config/states';
+import {
+  CompensationRateUnitsLabelsPlural,
+  CompensationRateUnitsLabelsSingular,
+  CompensationRateUnitsOptions,
+} from '../../../data-contracts/CompensationRateUnits';
+import { formatNumber } from '../../../utilities/valueFormatters';
+
+export function ApplicationCreatePage() {
+  const { state } = useConservationApplicationContext();
+  const navigate = useNavigate();
+
+  const navigateToEstimationToolPage = () => {
+    navigate(`/application/${state.conservationApplication.waterRightNativeId}/estimation`);
+  };
+
+  return (
+    <MapProvider>
+      <div className="application-create-page d-flex flex-column flex-grow-1 h-100">
+        <ApplicationNavbar
+          navigateBack={navigateToEstimationToolPage}
+          backButtonText="Back to Estimator"
+          centerText="Water Conservation Estimation Tool"
+        />
+
+        <ApplicationCreatePageForm />
+      </div>
+    </MapProvider>
+  );
+}
+
+const emptyStringPlaceholder = '';
+function ApplicationCreatePageForm() {
+  const { state } = useConservationApplicationContext();
+
+  // assumes all polygons are not intersecting
+  const acreageSum = state.conservationApplication.selectedMapPolygons.reduce(
+    (sum, polygon) => sum + polygon.acreage,
+    0,
+  );
+  const etAcreFeet = state.conservationApplication.polygonEtData.reduce(
+    (sum, polygon) => sum + polygon.averageYearlyEtInAcreFeet,
+    0,
+  );
+
+  return (
+    <div className="flex-grow-1 overflow-y-auto p-4">
+      <div className="mb-3">
+        <span className="fs-4 fw-bold">New Application</span>
+      </div>
+
+      <div className="d-flex gap-3 mb-4">
+        <span className="fw-bold">Water Right Native ID: {state.conservationApplication.waterRightNativeId}</span>
+
+        <span className="fw-bold">
+          Application ID: {state.conservationApplication.waterConservationApplicationDisplayId}
+        </span>
+
+        <span className="fw-bold">Funding Organization: {state.conservationApplication.fundingOrganizationName}</span>
+      </div>
+
+      <div className="mb-4">
+        <span>
+          Complete the below fields in order to submit your application to your state agency for verification. Be sure
+          everything is filled out accurately and truthfully.
+        </span>
+      </div>
+
+      <Form>
+        <FormSection title="Applicant Information">
+          <Form.Group className="col-4 mb-4">
```

Comment:
Probably worth making this a bit more responsive such as

```suggestion
          <Form.Group className="mb-2 col-xl-3 col-lg-4 col-md-6 col-sm-6">
```

Reply:
updated all the form groups with some standardized column breakpoints. here's a demo showcasing all of them. lmk what you think


https://github.com/user-attachments/assets/1d879f47-9329-4055-b801-24bb85737910